build/configs/artik053/netmgr/defconfig

```diff
@@ -656,7 +656,7 @@ CONFIG_NET_IPv6_PMTU_FOR_MULTICAST=y
 # Socket support
 #
 CONFIG_NET_SOCKET=y
-CONFIG_NBSDSOCKET_DESCRIPTORS=8
+CONFIG_NBSDSOCKET_DESCRIPTORS=16
```

Comment:
why did you change value?

Reply:
because at artik053 when i tried to join the wifi then alloc_socket() was called 9 times so for last socket it was returning -1.